test/test_helper.exs

```diff
@@ -96,13 +106,15 @@ defmodule TestRedis do
   def start do
     unless Config.get(:test_with_local_redis) == false do
       [] = :os.cmd('redis-server test/test-redis.conf')
-      :timer.sleep(100)
+      [] = :os.cmd('redis-server test/test-sentinel.conf --sentinel')
```

Comment:
Do we need to start Redis Sentinel for everything?  Maybe we should only start for those few tests?

Reply:
This is called only once at the beginning before any of the tests starts and stop is invoked at the end when all the tests are executed. Throughput wise delaying the start of sentinel server won't make any difference.

Reply:
Ah ok, my bad, makes sense.